src/core_atmosphere/physics/physics_mmm/bl_ysu.F

```diff
@@ -561,6 +561,7 @@ subroutine bl_ysu_run(ux,vx,tx,qvx,qcx,qix,nmix,qmix,p2d,p2di,pi2d,     &
      hgamt(i)  = 0.
      hgamq(i)  = 0.
      wscale(i) = 0.
+     we(i)     = 0.
```

Comment:
Should we omit changes in `bl_ysu.F` to keep this PR only for Noah-MP?

Reply:
I agree. I forgot to remove the initialization of "we" after finishing to test NoahMP.